storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -355,12 +362,30 @@
     @IsStringList
     public static final String TOPOLOGY_SPREAD_COMPONENTS = "topology.spread.components";
     /**
-     * The maximum number of states that will be searched looking for a solution in the constraint solver strategy.
+     * The maximum number of states that will be searched looking for a solution in resource aware strategies, e.g.
+     * in BaseResourceAwareStrategy.
      */
     @IsInteger
     @IsPositiveNumber
     public static final String TOPOLOGY_RAS_CONSTRAINT_MAX_STATE_SEARCH = "topology.ras.constraint.max.state.search";
     /**
+     * The maximum number of states that will be searched looking for a solution in resource aware strategies, e.g.
+     * in BaseResourceAwareStrategy. Backward compatibility config value for old topologies.
+     * @deprecated please use {@link Config#TOPOLOGY_RAS_CONSTRAINT_MAX_STATE_SEARCH}
+     */
+    @IsInteger
+    @IsPositiveNumber
+    @Deprecated
+    public static final String TOPOLOGY_RAS_CONSTRAINT_MAX_STATE_TRAVERSAL = "topology.ras.constraint.max.state.traversal";
```

Comment:
same comment on these....

Reply:
Will remove this also. No point adding a new deprecated variable.